CHANGELOG.md

```diff
@@ -2,6 +2,96 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.42.0] - 2018-12-24
+### Updated
+- Updated plotly.js to 1.43.0 [#423](https://github.com/plotly/dash-core-components/pull/423)
+
+### As part of [plotly.js 1.43.1](https://github.com/plotly/plotly.js/releases/tag/v1.43.1)
+
+### Fixed
+- Fix z-axis auto-type for cartesian + gl3d graphs (bug introduced in 1.43.0) [#3360]
```

Comment:
This isn't new in this PR, but... importing the whole plotly.js changelog en masse means that all of these links are broken, and ideally would need to be prefixed with `plotly/plotly.js`

Reply:
Opened #425 for that.